test_runner/fixtures/neon_fixtures.py

```diff
@@ -2981,3 +2995,20 @@ def fork_at_current_lsn(
     """
     current_lsn = pg.safe_psql("SELECT pg_current_wal_lsn()")[0][0]
     return env.neon_cli.create_branch(new_branch_name, ancestor_branch_name, tenant_id, current_lsn)
+
+
+def list_inuse_ports():
+    """
+    Print "netstat -tnlap" output to the test log. This is useful for debugging
+    port collisions in tests.
+    """
+    res = subprocess.run(
+        ["netstat", "-tnlap"],
```

Comment:
Note that `netstat` can have different flags in different linux distributions, and newer distros have `ss` instead of `netstat`. So we need to verify this works in CI.

Reply:
I added a comment noting that this is platform dependent and won't work everywhere. I think that's good enough.